client/src/components/ListItems/ListItems.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Item from "../Item/Item";
+import styles from "./ListItems.module.css";
+import PropTypes from "prop-types";
+import noop from "react-props-noop";
+const ListItem = ({
+  data,
+  removeIdFromDataHandler,
+  onChangeValueUpdateData,
+  setToastProps,
+  checkIfTextAlreadyExist,
+  displayToast,
+  setShowToast,
+  killToast,
+}) => {
+  return (
+    <ul className={styles.list}>
+      {data.map((item) => {
+        return (
+          <React.Fragment key={item.id}>
```

Comment:
Should move the key to the <Item> component instead.
Fragment is mainly used to wrap multiple child components without adding any container wrapper to the DOM, there is no point in wrapping each single item component with a fragment.